java/ql/test/library-tests/frameworks/apache-collections/Test.java

```diff
@@ -0,0 +1,990 @@
+package generatedtest;
+
+import java.util.Map;
+import java.util.ResourceBundle;
+import java.util.SortedMap;
+import java.util.TreeMap;
+import java.util.Vector;
+import org.apache.commons.collections4.Factory;
+import org.apache.commons.collections4.IterableMap;
+import org.apache.commons.collections4.IterableSortedMap;
+import org.apache.commons.collections4.KeyValue;
+import org.apache.commons.collections4.MapUtils;
+import org.apache.commons.collections4.MultiMap;
+import org.apache.commons.collections4.OrderedMap;
+import org.apache.commons.collections4.Transformer;
+import org.apache.commons.collections4.keyvalue.AbstractKeyValue;
+import org.apache.commons.collections4.keyvalue.AbstractMapEntry;
+import org.apache.commons.collections4.keyvalue.AbstractMapEntryDecorator;
+import org.apache.commons.collections4.keyvalue.DefaultKeyValue;
+import org.apache.commons.collections4.keyvalue.DefaultMapEntry;
+import org.apache.commons.collections4.keyvalue.TiedMapEntry;
+import org.apache.commons.collections4.keyvalue.UnmodifiableMapEntry;
+import org.apache.commons.collections4.map.MultiValueMap;
+
+//Test case generated by GenerateFlowTestCase.ql
+public class Test {
+
+	static Object getMapKey(AbstractKeyValue container) { return container.getKey(); }
+	static Object getMapKeyFromEntry(Map.Entry container) { return container.getKey(); }
+	static Object getMapKey(AbstractMapEntryDecorator container) { return container.getKey(); }
+	static Object getMapKey(Map container) { return container.keySet().iterator().next(); }
+	static Object getMapValue(AbstractKeyValue container) { return container.getValue(); }
+	static Object getMapValueFromEntry(Map.Entry container) { return container.getValue(); }
+	static Object getMapValue(AbstractMapEntryDecorator container) { return container.getValue(); }
+	static Object getMapValue(Map container) { return container.get(null); }
+
+	Object[] newWithArrayElement(Object element) { return new Object[] {element}; }
+	Object newWithElement(Object element) { Vector<String> v = new Vector<String>(); v.add((String)element); return v; }
```

Comment:
A `Vector` is an `Iterable` (and that's all you need this method for):
```suggestion
	Iterable<String> newWithElement(String element) { Vector<String> v = new Vector<String>(); v.add(element); return v; }
```